task/backend/bolt/bolt.go

```diff
@@ -173,6 +173,7 @@ func (s *Store) CreateTask(ctx context.Context, req backend.CreateTaskRequest) (
 
 func (s *Store) UpdateTask(ctx context.Context, req backend.UpdateTaskRequest) (backend.UpdateTaskResult, error) {
 	var res backend.UpdateTaskResult
+	fmt.Println("here1")
```

Comment:
I use these too!!  I sometimes get up to "here17". You might not want to leave it in though.

Reply:
DOH.